xtest/test_tdfs.py

```diff
@@ -83,3 +93,35 @@ def test_tdf_with_unbound_policy(encrypt_sdk, decrypt_sdk, pt_file, tmp_dir):
         assert False, "decrypt succeeded unexpectedly"
     except subprocess.CalledProcessError as exc:
         assert b"wrap" in exc.output
+
+
+def test_tdf_assertions(encrypt_sdk, decrypt_sdk, pt_file, tmp_dir):
+    if not tdfs.supports(encrypt_sdk, "assertions"):
+        pytest.skip(f"{encrypt_sdk} sdk doesn't yet support assertions")
+    if not tdfs.supports(decrypt_sdk, "assertions"):
+        pytest.skip(f"{decrypt_sdk} sdk doesn't yet support assertions")
+    ct_file = doEncryptWith(
+        pt_file,
+        encrypt_sdk,
+        "ztdf",
+        tmp_dir,
+        scenario="assertions",
+        az=[
+            assertions.Assertion(
+                id="assertion-1",
+                type="handling",
+                scope="payload",
+                appliesToState="encrypted",
+                statement=assertions.Statement(
+                    format="json",
+                    schema="https://schema.org/Person",
+                    value='{"name":"Alice"}',
```

Comment:
Can we update the schema and value to a more real-world scenario?`{"appliesToState":"encrypted","id":"424ff3a3-50ca-4f01-a2ae-ef851cd3cac0","scope":"tdo","statement":{"format":"json+stanag5636","schema":"urn:nato:stanag:5636:A:1:elements:json","value":"{\"ocl\":{\"pol\":\"62c76c68-d73d-4628-8ccc-4c1e18118c22\",\"cls\":\"SECRET\",\"catl\":[{\"type\":\"P\",\"name\":\"Releasable To\",\"vals\":[\"usa\"]}],\"dcr\":\"2024-10-21T20:47:36Z\"},\"context\":{\"@base\":\"urn:nato:stanag:5636:A:1:elements:json\"}}"},"type":"handling"}`

Reply:
switched